docker/tds/docker-compose.yml

```diff
@@ -20,25 +35,26 @@ services:
     ports:
       - "32843:8080"
     env_file: tds-docker.env
-  exam_mysql:
-    image: mysql
-    ports:
-      - "32777:3306"
-    volumes_from:
-      - exam_mysql_data
-    env_file: tds-docker.env
-  exam_mysql_data:
-    image: mysql
-    entrypoint: /bin/bash
   exam:
     image: fwsbac/tds-exam-service
     ports:
       - "80:8080"
     depends_on:
-      - assessment
-      - session
-      - student
-      - config
-      - exam_mysql
-    env_file:
-      - ./tds-docker.env
+      configuration-service:
+        condition: service_healthy
+      assessment:
+        condition: service_started
+      session:
+        condition: service_started
+      student:
+        condition: service_started
+      config:
+        condition: service_started
+    environment:
+      - CONFIG_SERVICE_URL=http://configuration-service:8888
+      - CONFIG_SERVICE_ENABLED=true
+      - SPRING_PROFILES_ACTIVE=local-docker
```

Comment:
Is `SPRING_PROFILES_ACTIVE` a built-in environment variable within docker compose that will pass it through when it starts up?  I'm asking because I was adding the new `cache-enabled` profile to our micro-services and was doing it with a new environment variable set in `tds-docker.env` and used in `docker-startup.sh` as you can see here: https://github.com/SmarterApp/TDS_ExamService/commit/c6861363fc43fc05b69bc7ee2684301d5cf6f5b7